app/src/main/java/com/android/sample/ui/camera/CameraPreview.kt

```diff
@@ -88,3 +88,19 @@ fun Carousel(openDialog: () -> Unit, itemsList: List<Bitmap>, deleteImage: (Bitm
     }
   }
 }
+
+@Composable
+fun CarouselNoModif(itemsList: List<Bitmap>) {
+  Row(modifier = Modifier.padding(8.dp).height(120.dp)) {
+    LazyRow {
+      items(itemsList.size) { bitmap ->
+        Card(modifier = Modifier.padding(4.dp)) {
+          Image(
+              bitmap = itemsList[bitmap].resize(100).asImageBitmap(),
+              contentDescription = "Selected Image",
+              modifier = Modifier.size(100.dp))
+        }
+      }
+    }
+  }
+}
```

Comment:
It is a good implementation, it has a very good UI.
But please try to remove the magic numbers